src/components/views/read/cvForm/types.ts

```diff
@@ -68,4 +68,5 @@ export type CVInfoObj = {
   portfolioInfo: PortfolioObj[];
   motivationInfo: MotivationObj;
   referencesInfo: ReferencesObj[];
+  [key: string]: any; // To access any object within this object.
```

Comment:
I see why you used that, maybe incorporating the changes I propose with the subkey type using the keyof operator would help you get rid of that widening type.